src/nidcpower/metadata/functions.py

```diff
@@ -883,6 +882,52 @@
         ],
         'returns': 'ViStatus'
     },
+    'FancyCreateAdvancedSequence': {
+        'codegen_method': 'python-only',
+        'returns': 'ViStatus',
+        'python_name': 'create_advanced_sequence',
+        'method_templates': [
+            { 'session_filename': 'fancy_advanced_sequence', 'documentation_filename': 'default_method', 'method_python_name_suffix': '', },
+        ],
+        'parameters': [
+            {
+                'direction': 'in',
+                'name': 'vi',
+                'type': 'ViSession',
+                'documentation': {
+                    'description': 'Identifies a particular instrument session. **vi** is obtained from the niDCPower_InitExtCal or niDCPower_InitializeWithChannels function.',
+                },
+            },
+            {
+                'direction': 'in',
+                'name': 'SequenceName',
+                'type': 'ViString',
+                'documentation': {
+                    'description': 'Name of sequence.',
+                },
+            },
+            {
+                'direction': 'in',
+                'name': 'Sequence',
+                'type': 'ViInt32[]',  # Doesn't matter here
+                'documentation': {
+                    'description': 'Sequence.',
+                },
+            },
+            {
+                'direction': 'in',
+                'name': 'setAsActiveSequence',
+                'type': 'ViBoolean',
+                'default_value': True,
+                'documentation': { 'description': 'Specifies that this current sequence is active.', },
+            },
+        ],
+        'documentation':
+        {
+            'description': 'Test.',
+            'note': 'Test.',
```

Comment:
Documentation is incomplete. If the intention is to merge this way and fix that in a separate PR, then add a TODO that we can point to an issue about it.

Reply:
Yes, documentation is complete. As stated before elsewhere, I am waiting to make sure that the approach is good before spending the time on documentation that may change.